### ISSUE TITLE
Getting an error when trying to select the Everything Toolbar from the Toolbar list 

### ISSUE THREAD

**Issue by kristian0808** [srwi/EverythingToolbar#384]
### Preflight Checklist

- [X] I have fully read the [README](https://github.com/srwi/EverythingToolbar/blob/master/README.md) and it did not solve the problem.
- [X] I have searched all [open and closed issues](https://github.com/srwi/EverythingToolbar/issues?q=) for the same bug report without success.

### EverythingToolbar Version

1.4.1.1023 (x64)

### Windows Version

Windows 10 21H2

### Steps to reproduce

1. Right-click on the Windows taskbar.
2. Go to the "Toolbars" option in the context menu that appears.
3. Select the "Everything" toolbar from the list of available toolbars.
4. Observe the error code or message that appears after selecting the "Everything Toolbar".

### Expected Behavior

I was trying to add the toolbar to my Windows taskbar by right-clicking on the taskbar and selecting it from the list of available toolbars. I expected to see the toolbar show up in the taskbar area after I selected it, but instead, an error code or message popped up.

### Actual Behavior

As I mentioned in steps to reproduce the problem in the 4th step I get this error code:

System.Windows.Markup.XamlParseException: The invocation of the constructor on type 'EverythingToolbar.Behaviors.ThemeAwareness' that matches the specified binding constraints threw an exception. ---> System.Runtime.InteropServices.COMException: The service cannot be started, either because it is disabled or because it has no enabled devices associated with it.

   at System.Management.ThreadDispatch.Start()
   at System.Management.ManagementScope.Initialize()
   at System.Management.ManagementEventWatcher.Initialize()
   at System.Management.ManagementEventWatcher.Start()
   at EverythingToolbar.Helpers.RegistryWatcher..ctor(RegistryEntry target) in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar\Helpers\RegistryWatcher.cs:line 45
   at EverythingToolbar.Behaviors.ThemeAwareness..ctor() in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar\Behaviors\ThemeAwareness.cs:line 60
   --- End of inner exception stack trace ---
   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at EverythingToolbar.ToolbarControl..ctor() in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar.Deskband\ToolbarControl.xaml.cs:line 20
   at CSDeskBand.Deskband..ctor() in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar.Deskband\Deskband.cs:line 24

### Log output

System.Windows.Markup.XamlParseException: The invocation of the constructor on type 'EverythingToolbar.Behaviors.ThemeAwareness' that matches the specified binding constraints threw an exception. ---> System.Runtime.InteropServices.COMException: The service cannot be started, either because it is disabled or because it has no enabled devices associated with it.

   at System.Management.ThreadDispatch.Start()
   at System.Management.ManagementScope.Initialize()
   at System.Management.ManagementEventWatcher.Initialize()
   at System.Management.ManagementEventWatcher.Start()
   at EverythingToolbar.Helpers.RegistryWatcher..ctor(RegistryEntry target) in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar\Helpers\RegistryWatcher.cs:line 45
   at EverythingToolbar.Behaviors.ThemeAwareness..ctor() in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar\Behaviors\ThemeAwareness.cs:line 60
   --- End of inner exception stack trace ---
   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at EverythingToolbar.ToolbarControl..ctor() in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar.Deskband\ToolbarControl.xaml.cs:line 20
   at CSDeskBand.Deskband..ctor() in C:\Users\Stephan\Entwicklung\EverythingToolbar\EverythingToolbar.Deskband\Deskband.cs:line 24

**Comment by kristian0808** [comment]
@srwi

**Comment by srwi** [comment]
> @srwi

Hello @kristian0808, please refrain from pushing issues like that. I saw your message and will take a look when I get the time.

**Comment by srwi** [comment]
Most likely you have disabled a Windows service called "Windows Management Instrumentation". I can reproduce the error message by setting its startup type to "Disabled".

To reenable it, type the command `services.msc` into the start menu and locate the mentioned service in the list. Open its properties by right clicking it and set the startup type to automatic. After a reboot I would expect EverythingToolbar to work as expected.

Please let me know if that works out for you!

**Comment by kristian0808** [comment]
Hello @srwi, thank you for your help in resolving the issue. I checked the service you mentioned and found that it was already running automatically. However, I noticed that the 'Windows Management Service' was disabled and set to manual. After starting the service, I tried again and the issue was resolved. I appreciate the fantastic work you've done. As a newcomer to Github, I apologize if I seemed rushed in my previous message. Best regards!
